src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProject_IWorkspaceProjectContext.cs

```diff
@@ -23,12 +25,12 @@ internal sealed partial class CPSProject : IWorkspaceProjectContext
         /// The <see cref="VisualStudioProjectOptionsProcessor"/> we're using to parse command line options. Null if we don't
         /// have the ability to parse command line options.
         /// </summary>
-        private readonly VisualStudioProjectOptionsProcessor _visualStudioProjectOptionsProcessor;
+        private readonly VisualStudioProjectOptionsProcessor? _visualStudioProjectOptionsProcessor;
 
         private readonly VisualStudioWorkspaceImpl _visualStudioWorkspace;
         private readonly IProjectCodeModel _projectCodeModel;
-        private readonly Lazy<ProjectExternalErrorReporter> _externalErrorReporterOpt;
-        private readonly EditAndContinue.VsENCRebuildableProjectImpl _editAndContinueProject;
+        private readonly Lazy<ProjectExternalErrorReporter?> _externalErrorReporterOpt;
```

Comment:
```suggestion
        private readonly Lazy<ProjectExternalErrorReporter?> _externalErrorReporter;
```

Reply:
I chose to skip this _for now_: this class is partial and there are uses in the other parts that aren't annotated yet, and I didn't want to annotate more than necessary if we're taking this through QB.

Reply:
Actually maybe doing that one file won't be a problem either.

Reply:
Fixed.